about.html

```diff
@@ -0,0 +1,177 @@
+                    <img src="/images-capstone/web-summit-logo.png" alt="about me section logo"/>
+        </div>
+        <p class="about-main-content">Lorem ipsum dolor sit amet consectetur adipisicing elit:</p>
+        <p><a href="mailto:websummitlisbon@gmail.com" class="about-email">websummitlisbon@gmail.com</a></p>
+    </section>
+    <!--end of about info section-->
+    <!--event partners section-->
+    <section id="partners">
+        <div id="partners-container">
+            <h2 id="partners-title"> Partners </h2>
+            <hr>
+            <div class="partner-logos">
+                <ul>
+                    <li class="brand-logo"><i class="fa-brands fa-apple-pay"></i></li>
+                    <li class="brand-logo"><i class="fa-brands fa-google"></i></li>
+                    <li class="brand-logo"><i class="fa-brands fa-uber"></i></li>
+                    <li class="brand-logo"><i class="fa-brands fa-airbnb"></i></li>
+                    <li class="brand-logo"><i class="fa-brands fa-microsoft"></i></li>
+                </ul>
+            </div>
+        </div>
+    </section>
+    <!--end of event partners section-->
+        <!--past events section-->
+<section id="past-events">
+    <div id="past-events-container">
+        <h2 class="see-past-events">See past CC Global Summits events</h2>
+        <hr>
+        <p>Check previous events and summits</p>
+        <ul id="events">
+            <li>
+                <!-- <div class="overlay-img"> -->
+                <img class="previous-web-summit" src="/images-capstone/web-summit-2021.png" alt="web summit 2021">
+                    <div class="overlay">
+                        <h3>2021</h3>
+                        <p>Web Summit Rio de Janeiro</p>
+                    </div>
+            <!-- </div> -->
+            </li>
+
+            <li>
+                <!-- <div class="overlay-img"> -->
+                <img class="previous-web-summit" src="/images-capstone/web-summit-2020.png" alt="web summit 2020">
+                    <div class="overlay">
+                        <h3>2020</h3>
+                        <p>Web Summit Toronto</p>
+                    </div>
+            <!-- </div> -->
+            </li>
+        </ul>
+    </div>
+</section>
+        <!--end of past events section-->
+    <footer id="second-copyright">
+        <div id="second-footer-container">
+            <div class="second-footer-logo">
+                <img src="images-capstone/creative-commons-logo.png" alt="creative commons" id="cc-logo-black"/>
+                <img src="images-capstone/cc-logo-white.png" alt="Creative Commons" id="cc-logo-white">
+            </div>
+            <div class="second-cc-info">
+                <p><a rel="license" href="http://creativecommons.org/licenses/by-nc/4.0/"><img
+                            alt="Creative Commons License"
+                            src="https://i.creativecommons.org/l/by-nc/4.0/88x31.png" /></a></p>
```

Comment:
- [ ] As required by the previous reviewer about the overlapping text issue, I noticed that it was not fixed correctly, in mobile it is okay but on the Desktop the issue is still there. And it is not the only place with the issue of overlapping elements. I think the issue is from your styling, when we move from the Big screen to medium the issue appears on many elements of your pages.
#### [Please click here to see some overlapping issues of your project](https://photos.google.com/share/AF1QipMpxU0n9V7zXBCzkqrGzoch4FP2NvQOE5KlQ7VLtaTOt6g709G4qcACZm5XlE1voQ/photo/AF1QipP9otDvNq1BwTwhxkKge9prqfKp_8G2_Ig-6xgu?key=RE9wSVhxZFk5elpRWGpHQ1pMVE9teUpXWHlWcDJR)

- [ **OPTIONAL** ] **RECOMENDED**: MAke sure you remove the extra space at the right and scrolling at the bottom so style your page correctly to match the provided template.